README.md

```diff
@@ -169,14 +172,31 @@ The command above will create three pods associated to the existing `imagestream
 objects that point to `openstack`, `openshift` and `kubevirt` must-gather container
 images.
 
-```
+```bash
 [must-gather] OUT pod for plug-in image quay.io/openstack-k8s-operators/openstack-must-gather:latest created
 [must-gather] OUT pod for plug-in image quay.io/kubevirt/must-gather@sha256:501e30ac7d5b9840a918bb9e5aa830686288ccfeee37d70aaf99cd2e302a2bb0 created
 [must-gather] OUT pod for plug-in image quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:e9601b492cbb375f0a05310efa6025691f8bba6a97667976cd4baf4adf0f244c created
 ...
 ...
 ```
 
+### EDPM compute must gather via `sos` tool
+
+To gather config files and logs from the deployed compute nodes, we need to rely on the `sos` tool (former `sosreport`).
+
+There's an open [PR](https://github.com/sosreport/sos/pull/3414) to enable path gathering from EDPM deployed compute nodes so if you want to test it on running nodes, you need to follow these steps on a running compute node:
```

Comment:
I would rather add a "Depends-On" (a logical depends on given we don't have any CI here), and wait until the patch [1] merges.
When [1] merges, we can change this statement to describe how to run a standalone `sos report` on an existing compute node (which is basically described below).

[1]  https://github.com/sosreport/sos/pull/3414

Reply:
Totally agree on this.